doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,423 @@
+2. convert it to a numerical function using {func}`~.lambdify`
+3. solve the initial value problem by [numerically integrating the ODE using
+   SciPy's
+   `solve_ivp`](https://docs.scipy.org/doc/scipy/reference/integrate.html#solving-initial-value-problems-for-ode-systems).
+
+Here is an [example from the field of chemical
+kinetics](https://www.sympy.org/scipy-2017-codegen-tutorial/notebooks/25-chemical-kinetics-intro.html)
+where the nonlinear ordinary differential equations take this form:
+
+$$ r_f = & k_f y_0(t)^2  y_1(t) \\
+r_b = & k_b y_2(t)^2 \\
+\frac{d y_0(t)}{dt} = & 2(r_b - r_f) \\
+\frac{d y_1(t)}{dt} = & r_b - r_f \\
+\frac{d y_2(t)}{dt} = & 2(r_f - r_b) $$
+
+and
+
+$$
+
+\vec{y}(t) = \begin{bmatrix} y_0(t) \\
+y_1(t) \\
+y_2(t) \end{bmatrix} $$
+
+```{eval-rst}
+.. plot::
+  :format: doctest
+  :include-source: True
+  :context: close-figs
+
+  >>> from sympy import symbols, lambdify
+  >>> import numpy as np
+  >>> import scipy.integrate
+  >>> import matplotlib.pyplot as plt
+  >>> # Create symbols y0, y1, and y2
+  >>> y = symbols('y:3')
+  >>> kf, kb = symbols('kf kb')
+  >>> rf = kf * y[0]**2 * y[1]
+  >>> rb = kb * y[2]**2
+  >>> # Derivative of the function y(t); values for the three chemical species
+  >>> # for input values y, kf, and kb
+  >>> ydot = [2*(rb - rf), rb - rf, 2*(rf - rb)]
+  >>> ydot
+  [2*kb*y2**2 - 2*kf*y0**2*y1, kb*y2**2 - kf*y0**2*y1, -2*kb*y2**2 + 2*kf*y0**2*y1]
+  >>> t = symbols('t') # not used in this case
+  >>> # Convert the SymPy symbolic expression for ydot into a form that
+  >>> # SciPy can evaluate numerically, f
+  >>> f = lambdify((t, y, kf, kb), ydot)
+  >>> k_vals = np.array([0.42, 0.17]) # arbitrary in this case
+  >>> y0 = [1, 1, 0] # initial condition (initial values)
+  >>> t_eval = np.linspace(0, 10, 50) # evaluate integral from t = 0-10 for 50 points
+  >>> # Call SciPy's ODE initial value problem solver solve_ivp by passing it
+  >>> #   the function f,
+  >>> #   the interval of integration,
+  >>> #   the initial state, and
+  >>> #   the arguments to pass to the function f
+  >>> solution = scipy.integrate.solve_ivp(f, (0, 10), y0, t_eval=t_eval, args=k_vals)
+  >>> # Extract the y (concentration) values from SciPy solution result
+  >>> y = solution.y
+  >>> # Plot the result graphically using matplotlib
+  >>> plt.plot(t_eval, y.T) # doctest: +SKIP
+  >>> # Add title, legend, and axis labels to the plot
+  >>> plt.title('Chemical Kinetics') # doctest: +SKIP
+  >>> plt.legend(['NO', 'Br$_2$', 'NOBr'], shadow=True) # doctest: +SKIP
+  >>> plt.xlabel('time') # doctest: +SKIP
+  >>> plt.ylabel('concentration') # doctest: +SKIP
+  >>> # Finally, display the annotated plot
+  >>> plt.show()
+  ```
+
+SciPy's `solve_ivp` returns a result containing `y` (numerical function result,
+here, concentration) values for each of the three chemical species,
+corresponding to the time points `t_eval`.
+
+## Ordinary Differential Equation Solving Hints
+
+### Return Unevaluated Integrals
```

Comment:
We really just need to add this as a flag to dsolve so you don't have to mess with hints to do this.

Reply:
That would be nice.